README.md

```diff
@@ -1,7 +1,32 @@
-# Config File Validator
+<div align="center">
+<img src="./img/logo.png" width="200" height="200"/>
+<h1>Config File Validator</h1>
+</div>
+
+<p align="center">
+  <a href="https://opensource.org/licenses/Apache-2.0">
+  <img src="https://img.shields.io/badge/License-Apache_2.0-blue.svg" alt="Apache 2 License">
+  </a>
+
+  <a href="https://pkg.go.dev/github.com/Boeing/config-file-validator">
+  <img src="https://pkg.go.dev/badge/github.com/Boeing/config-file-validator.svg" alt="Go Reference">
+  </a>
+
+  <a href="https://goreportcard.com/report/github.com/Boeing/config-file-validator">
+  <img src="https://goreportcard.com/badge/github.com/Boeing/config-file-validator" alt="Go Report Card">
+  </a>
+
+  <a href="https://github.com/boeing/config-file-validator/actions/workflows/go.yml">
+  <img src="https://github.com/boeing/config-file-validator/actions/workflows/go.yml/badge.svg" alt="Pipeline Status">
+  </a>
+  
+  <a href='https://coveralls.io/github/kehoecj/config-file-validator?branch=readme_updates'>
+    <img src='https://coveralls.io/repos/github/kehoecj/config-file-validator/badge.svg?branch=readme_updates' alt='Coverage Status' />
```

Comment:
This is tied to your repo and specific branch. Assuming this should be updated to point to the boeing repo and mainline branch.

Reply:
Put in a request to the Boeing org to allow access to coveralls - if rejected I may need to leave the badge out for now and pursue a different path 🤞

Reply:
Looks like approval will take a while. I removed the coverage badge and job and will re-add after approval